packages/core/src/base/cacheManagerKeyv.ts

```diff
@@ -0,0 +1,277 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+"use strict";
+
+import fetch = require("minipass-fetch");
+import Keyv = require("keyv");
+import ssri = require("ssri");
+import url = require("url");
+
+import { ICacheManager } from "./cacheManager";
+
+/**
+ * Calculate a reasonable time to live for the response based on the request and response headers.
+ *
+ * @param response The response to calculate the TTL for
+ */
+const getTTL = (response: fetch.Response): number => {
```

Comment:
Not camelCase.  Only first character in the abbreviation should be capitalized as our agreed upon camel case styling 
For reference: https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

Reply:
Fixed